mempool/src/pool/mod.rs

```diff
@@ -1020,7 +1024,8 @@ impl<M: MemoryUsageEstimator> Mempool<M> {
                         })
                 },
             )
-            .map(|feerate| std::cmp::max(feerate, INCREMENTAL_RELAY_FEE_RATE))
+            // TODO: should INCREMENTAL_RELAY_FEE_RATE also be taken into account here?
```

Comment:
@iljakuklic Could you please double-check this? This seems correct to me, but I'm not sure whether I'm missing something.

Reply:
Looks fine to me